README.md

```diff
@@ -29,7 +29,6 @@ fixes and constructive feedback.
 * [Get support](https://discourse.charmhub.io/)
 * [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
 * [Contribute](https://charmhub.io/discourse-k8s/docs/contributing)
-* [Roadmap](https://charmhub.io/discourse-k8s/docs/roadmap)
```

Comment:
Why removing the link to the Roadmap from here?

Reply:
It doesn't exist anymore